tests/utils/check_code.sh

```diff
@@ -57,6 +57,13 @@ else
 fi
 
 path=$2
+if [[ -d "./geoips_dev_utils/.config" ]]; then
+    # This is for GitHub Actions, geoips repo cloned as "geoips_dev_utils"
+    CONFIG_PATH=./geoips_dev_utils/.config
+else
+    # This is for local development clone
+    CONFIG_PATH=$GEOIPS_PACKAGES_DIR/geoips/.config
+fi
```

Comment:
@AJThorpe Perhaps we leave them in the .config directory, update this to reference either $GEOIPS_PACKAGES_DIR/geoips (local development checkout) or geoips_dev_utils/.config (github actions checkout), and call it a day?  So, it will not be automatically referenced from flake8 without passing the path to the config file - but that may be ok (basically we are giving the people the tools to allow them to call flake8/black with our settings, but not forcing the repo to do it by default ?)

Reply:
@jsolbrig Thoughts on how else we could handle this for github actions vs local clone ?

Reply:
Could we just find the config directory this way?

```bash
SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
GEOIPS_DIR=$(dirname "$(dirname $SCRIPT_DIR)")
CONFIG_DIR="${GEOIPS_DIR}/.config"
```

Reply:
Oh yeah,  that should work,  it will always be relative to the check code script that is currently running...

Reply:
Used `dirname $0`/../../.config